examples/importers/utrade.py

```diff
@@ -122,9 +123,9 @@ def extract(self, filepath, existing):
                     rate = D(match.group(3))
 
                     if rtype == 'BUY':
-                        cost = position.Cost(rate, self.currency, None, None)
+                        cost = position.CostSpec(rate, None, self.currency, None, None, None)
```

Comment:
Why? `Cost` should be just fine.

Reply:
a Cost requires a date (the booking code will insert it for all postings, falling back to the entry date)